### ISSUE TITLE
🐛 [BUG] - alt, aria-label, title, rel, target attributes related Issues in Some Files

### ISSUE THREAD

**Issue by NarendraKoya999** [AKD-01/blogweet#574]
## 🐛 Bug Report Form

|                |                     |
|----------------|---------------------|
| **Description** | `We need for adding alt, target, title, rel, aria-label attributes in Some Files for Screen Readers and Accessibility Point of View` |
| **Reproduction URL** | https://blogweet.vercel.app/ |
| **Reproduction Steps** | 1. Step 1...<br>2. Step 2...<br>3. Step 3...<br>4. Step 4... |
| **Screenshots** | ![Screenshot 1](screenshot1.png) |
| **Logs** | Relevant log output goes here |
| **Additional Context** |
1. The `aria-label attribute` provides a way to place a descriptive text label on an object, such as a link, when there are no elements visible on the page that describe the object. If descriptive elements are visible on the page, the aria-labelledby attribute should be used instead of aria-label
2. The `rel attribute` specifies the relationship between the current document and the linked document. Only used if the href attribute is present
3. The `title attribute` gives users an important indication as to where they will be directed to when they click on a link. If the user hovers with the mouse over a link reference, the title text, which was defined in the attribute, will be displayed 
4. The` target attribute` specifies where the linked document will open when the link is clicked. The default is the current window. If target="_blank", the linked document will open in a new tab or (on older browsers) a new window.
5. The required `alt attribute` specifies an alternate text for an image, if the image cannot be displayed. The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader) .|

🌐 **Browsers**
Which browsers are you experiencing the problem on? (Select all that apply)

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Microsoft Edge
- [ ] Opera
- [ ] Brave

💻 **OS**
What is the impacted environment (operating system)? (Select all that apply)

- [x] Windows
- [ ] Linux
- [ ] Mac
- [ ] iOS
- [ ] Android

📝 **Sections Completed**
Check the box of the sections you have completed:

- [x] Description
- [x] Reproduction URL
- [ ] Reproduction Steps
- [ ] Screenshots
- [ ] Logs
- [x] Browsers
- [x] OS
- [x] Additional Context

🔍 **Priority**
Please select the priority of the bug:
- [ ] High
- [x] Medium
- [ ] Low

📜 **Code of Conduct**
By submitting this issue, you agree to follow our Code of Conduct:

- [x] I follow Contributing Guidelines of this project.
- [x] I have checked the current issues for duplicate problems.
- [x] I have read the readme.

**Comment by NarendraKoya999** [comment]
@AKD-01 . Please review on my Issue as soon as Possible
